index.d.ts

```diff
@@ -1,7 +1,18 @@
+export type Options = {
+	/**
+	Modifies the output style.
+	If false returns value in recompilable source form.
+
```

Comment:
Would be useful to link to the osascript docs about the different forms.

Reply:
Agree. I added the link to osascript's man page in [da05ff8](https://github.com/sindresorhus/run-applescript/pull/14/commits/da05ff82061133e87977cfc2af0782a18c46356f)